static/app/views/settings/organizationTeams/teamMembers.tsx

```diff
@@ -22,6 +22,7 @@ import LoadingIndicator from 'sentry/components/loadingIndicator';
 import Pagination from 'sentry/components/pagination';
 import Panel from 'sentry/components/panels/panel';
 import PanelHeader from 'sentry/components/panels/panelHeader';
+import {Flex} from 'sentry/components/profiling/flex';
```

Comment:
not sure how I feel about importing something from `profiling` into here

Reply:
we should move `Flex` out of the profiling folder, we use it everywhere in feedback too!

Reply:
I'll move it in another PR. A few files will get touched in profiling, replay and feedback.